Adds loading indicator while fetching history

madsnedergaard opened the pull request:
## Description

Makes it clear to users that data is being fetched, instead of first showing "no data is available" until data for a zone is loaded.

### Before
![Screenshot 2022-03-18 at 11 47 25](https://user-images.githubusercontent.com/3296643/158989573-3609755b-4807-4ddd-ba18-e106faac7835.png)

### After
![Screenshot 2022-03-18 at 11 46 54](https://user-images.githubusercontent.com/3296643/158989579-befe418b-87f0-4db8-ae88-26cdbc8af24e.png)

Kongkille reviewed (approved):
This is great.

I am wondering if we'd want to still show the name of the zone while loading? I think it's nice small feedback that the users know they have clicked the right zone and is now awaiting the detailed data. 

(I realize it's less than a second usually) 

Adds a bit of extra work, so regardless we could address that in another PR

madsnedergaard commented:
Good point @Kongkille, I have changed it to this now:

<img width="586" alt="Screenshot 2022-03-25 at 12 51 39" src="https://user-images.githubusercontent.com/3296643/160115891-37046e29-9ea8-4640-97f5-eb374c5103bd.png">

I also fixed the back-arrow button being white on hover so you couldn't see it 🤔

Kongkille reviewed (approved):
Excellent!
